modules/utxo-lib/src/bitgo/PsbtUtil.ts

```diff
@@ -0,0 +1,81 @@
+import { decodeProprietaryKey } from 'bip174/src/lib/proprietaryKeyVal';
+import {
+  ProprietaryKeySearch,
+  ProprietaryKeySubtype,
+  ProprietaryKeyValue,
+  PSBT_PROPRIETARY_IDENTIFIER,
```

Comment:
You can probably move these into this file

Reply:
Done